src/main/java/com/google/devtools/build/lib/skyframe/SkyFunctions.java

```diff
@@ -158,6 +158,7 @@ public final class SkyFunctions {
       SkyFunctionName.createHermetic("BAZEL_DEP_GRAPH");
   public static final SkyFunctionName BAZEL_LOCK_FILE =
       SkyFunctionName.createHermetic("BAZEL_LOCK_FILE");
+  public static final SkyFunctionName REPO_SPEC = SkyFunctionName.createNonHermetic("REPO_SPEC");
```

Comment:
Just an observation: I noticed that `BAZEL_MODULE_RESOLUTION` was marked hermetic even though it probably wasn't before this PR (due to the registry fetches).

Reply:
ah, that does look like an oversight. However in practice this shouldn't matter much since we assume existing registry contents are immutable anyway.